packages/web-twig/src/Resources/components/Tabs/TabLink.twig

```diff
@@ -11,32 +11,33 @@
 {%- set _rootClassName = _spiritClassPrefix ~ 'Tabs__link' -%}
 
 {# Attributes #}
-{%- set _ariaControlsAttr = _target ? 'aria-controls=' ~ _target : null -%}
-{%- set _dataTargetAttr = _target ? 'data-target=#' ~ _target : null -%}
-{%- set _dataToggleAttr = _href ? null : 'data-toggle=tabs' -%}
-{%- set _hrefAttr = _href ? 'href=' ~ _href : null -%}
-{%- set _idAttr = _id ? ' id=' ~ _id : null -%}
-{%- set _roleAttr = _href ? 'role=tab' : null -%}
-{%- set _typeAttr = _href ? null : 'type=button' -%}
+{%- set _ariaControlsAttr = _target ? 'aria-controls="' ~ _target | escape('html_attr') ~ '"' : null -%}
+{%- set _dataTargetAttr = _target ? 'data-target="#' ~ _target | escape('html_attr') ~ '"' : null -%}
+{%- set _dataToggleAttr = _href ? null : 'data-toggle="tabs"' -%}
+{%- set _hrefAttr = _href ? 'href="' ~ _href | escape('html_attr') ~ '"' : null -%}
+{%- set _idAttr = _id ? ' id="' ~ _id | escape('html_attr') ~ '"' : null -%}
+{%- set _roleAttr = _href ? 'role="tab"' : null -%}
+{%- set _typeAttr = _href ? null : 'type="button"' -%}
 
 {# Miscellaneous #}
 {%- set _styleProps = useStyleProps(props) -%}
 {%- set _ariaSelected = _isSelected ? 'true' : 'false' -%}
 {%- set _classNames = [ _rootClassName, _isSelectedClassName, _styleProps.className ] -%}
 {%- set _elementType = _href ? 'a' : 'button' -%}
+{%- set _mainPropsWithoutId = props | filter((value, prop) => prop != 'id') -%}
```

Comment:
maybe we should update this everywhere so we can add another filter without renaming the variable too

Reply:
but it could be another issue